src/redux/books/books.js

```diff
@@ -1,67 +1,60 @@
-import { v4 as uuidv4 } from 'uuid';
+/* eslint-disable camelcase */
+import { createAsyncThunk } from '@reduxjs/toolkit';
 
+const Api = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/3NpzSS6TCHlgPC9vqAsu/books';
+
+// Actions
 const ADD = 'bookstore/books/addBook';
 const REMOVE = 'bookstore/books/removeBook';
+const GET_BOOKS = 'bookstore/books/fetchBooks';
 
-const initialState = {
-  books: [
-    {
-      id: uuidv4(),
-      category: 'action',
-      title: 'Titanic',
-      author: 'Winfred',
-    },
-    {
-      id: uuidv4(),
-      category: 'action',
-      title: 'Titanic',
-      author: 'Winnie',
-    },
-    {
-      id: uuidv4(),
-      category: 'action',
-      title: 'tanic',
-      author: 'Winfred',
-    },
-    {
-      id: uuidv4(),
-      category: 'action',
-      title: 'Titan',
-      author: 'Winfred',
-    },
-  ],
-};
+const initialState = [];
 
+// Add Reducers
 export default function booksReducer(state = initialState, action) {
   switch (action.type) {
-    case ADD: {
-      const newbook = [...state.books, action.payload];
-      return { ...state.books, books: newbook };
-    }
+    case `${ADD}/fulfilled`:
+      return action.payload;
```

Comment:
:red_circle: As you can see here you return a new state this won't allow you to access the old state. You are supposed to add the current book to the previous state. You shouldn't mutate the global state variable. Instead of this, you should add the new book to the current list of books as follow :arrow_down: 

`[....state, action.payload]`